### DISCUSSION TITLE
GitHub Actions Automerge workflow disabled?

### DISCUSSION THREAD

**Discussion by apparition47** [Homebrew/discussions#1086] [Casks] [answered]
Workflow here: https://github.com/Homebrew/homebrew-cask/actions/workflows/automerge.yml

This workflow seems to have been disabled recently preventing [my `brew bump-cask-pr` from getting automerged](https://github.com/Homebrew/homebrew-cask/pull/101433/files). Any idea why?

**Comment by SMillerDev** [comment, marked as answer]
We found a potential vulnerability in it and disabled it until we make sure it's remedied. Until then you'll have to wait for a human to merge your work.
